Sources/SmileID/Classes/DocumentVerification/Model/DocumentCaptureModel.swift

```diff
@@ -333,12 +341,49 @@ class DocumentCaptureViewModel: ObservableObject, JobSubmittable, ConfirmationDi
             guard let self else {
```

Comment:
Can `self` ever be `nil`?

Reply:
Unlikely, but possible if the viewmodel is deallocated before the DispatchQueue executes this block.